src/test/kotlin/org/reactome/cli/ReactomeCliTest.kt

```diff
@@ -0,0 +1,17 @@
\",\"dbId\":76005,\"name\":\"Response to elevated platelet cytosolic Ca2+\",\"species\":{\"dbId\":48887,\"taxId\":\"9606\",\"name\":\"Homo sapiens\"},\"llp\":true,\"entities\":{\"resource\":\"TOTAL\",\"total\":148,\"found\":1,\"ratio\":0.009384908053265695,\"pValue\":0.00938490805326575,\"fdr\":0.00938490805326575,\"exp\":[]},\"reactions\":{\"resource\":\"TOTAL\",\"total\":14,\"found\":1,\"ratio\":9.173710766004849E-4},\"inDisease\":false},{\"stId\":\"R-HSA-1474228\",\"dbId\":1474228,\"name\":\"Degradation of the extracellular matrix\",\"species\":{\"dbId\":48887,\"taxId\":\"9606\",\"name\":\"Homo sapiens\"},\"llp\":true,\"entities\":{\"resource\":\"TOTAL\",\"total\":148,\"found\":1,\"ratio\":0.009384908053265695,\"pValue\":0.00938490805326575,\"fdr\":0.00938490805326575,\"exp\":[]},\"reactions\":{\"resource\":\"TOTAL\",\"total\":105,\"found\":1,\"ratio\":0.006880283074503637},\"inDisease\":false},{\"stId\":\"R-HSA-76002\",\"dbId\":76002,\"name\":\"Platelet activation, signaling and aggregation\",\"species\":{\"dbId\":48887,\"taxId\":\"9606\",\"name\":\"Homo sapiens\"},\"llp\":false,\"entities\":{\"resource\":\"TOTAL\",\"total\":293,\"found\":1,\"ratio\":0.018579581483830057,\"pValue\":0.0185795814838301,\"fdr\":0.0185795814838301,\"exp\":[]},\"reactions\":{\"resource\":\"TOTAL\",\"total\":119,\"found\":1,\"ratio\":0.007797654151104122},\"inDisease\":false},{\"stId\":\"R-HSA-1474244\",\"dbId\":1474244,\"name\":\"Extracellular matrix organization\",\"species\":{\"dbId\":48887,\"taxId\":\"9606\",\"name\":\"Homo sapiens\"},\"llp\":true,\"entities\":{\"resource\":\"TOTAL\",\"total\":350,\"found\":1,\"ratio\":0.02219403931515536,\"pValue\":0.0221940393151554,\"fdr\":0.0221940393151554,\"exp\":[]},\"reactions\":{\"resource\":\"TOTAL\",\"total\":330,\"found\":1,\"ratio\":0.021623746805582857},\"inDisease\":false},{\"stId\":\"R-HSA-109582\",\"dbId\":109582,\"name\":\"Hemostasis\",\"species\":{\"dbId\":48887,\"taxId\":\"9606\",\"name\":\"Homo sapiens\"},\"llp\":false,\"entities\":{\"resource\":\"TOTAL\",\"total\":804,\"found\":1,\"ratio\":0.05098287888395688,\"pValue\":0.05098287888395692,\"fdr\":0.05098287888395692,\"exp\":[]},\"reactions\":{\"resource\":\"TOTAL\",\"total\":342,\"found\":2,\"ratio\":0.022410064871240417},\"inDisease\":false},{\"stId\":\"R-HSA-382551\",\"dbId\":382551,\"name\":\"Transport of small molecules\",\"species\":{\"dbId\":48887,\"taxId\":\"9606\",\"name\":\"Homo sapiens\"},\"llp\":false,\"entities\":{\"resource\":\"TOTAL\",\"total\":1002,\"found\":1,\"ratio\":0.06353836398224477,\"pValue\":0.06353836398224477,\"fdr\":0.06353836398224477,\"exp\":[]},\"reactions\":{\"resource\":\"TOTAL\",\"total\":465,\"found\":1,\"ratio\":0.030469825044230393},\"inDisease\":false}],\"resourceSummary\":[{\"resource\":\"TOTAL\",\"pathways\":12,\"filtered\":12},{\"resource\":\"UNIPROT\",\"pathways\":12,\"filtered\":12}],\"speciesSummary\":[{\"dbId\":48887,\"taxId\":\"9606\",\"name\":\"Homo sapiens\",\"pathways\":12,\"filtered\":12}],\"warnings\":[]}"
```

Comment:
Just to note, the token may not always be valid.  We clear out the token cache on each release and so it will expire after at most 3 months.

Reply:
Indeed we won't keep this assertion for long, was mostly just to have something there. maybe just isNotNull() would be more appropriate.